ladybug/sunpath.py

```diff
@@ -151,36 +151,19 @@ def calculateSunFromDataTime(self, datetime, isSolarTime=False):
         hourAngle = (solTime * 15 + 180) if (solTime * 15 < 0) \
             else (solTime * 15 - 180)
 
-        # RADIANS
-        zenith = math.acos(math.sin(self._latitude) * math.sin(solDec) +
-                           math.cos(self._latitude) * math.cos(solDec) *
-                           math.cos(math.radians(hourAngle)))
-
-        altitude = (math.pi / 2) - zenith
-
-        if hourAngle == 0.0 or hourAngle == -180.0 or hourAngle == 180.0:
+        # Degrees
+        zenith = math.degrees(math.acos(math.sin(self._latitude)*math.sin(math.radians(solDec))+math.cos(self._latitude)*math.cos(math.radians(solDec))*math.cos(math.radians(hourAngle))))
```

Comment:
1. Add a white space before and after * - / +
2. Break down the line into smaller segments. We us 89 characters.